README.md

```diff
@@ -3,13 +3,13 @@
 [![Build Status](https://travis-ci.com/paralleldrive/react-feature-toggles.svg?token=Ba8H1FN3UT5CqqFhs2AM&branch=master)](https://travis-ci.com/paralleldrive/react-feature-toggles)
 
 ## Version 2
-This is version 2, it contains many breaking changes from version 1. 
+
+This is version 2, it contains many breaking changes from version 1.
 
 ## Requirements
 
 React Feature Toggles attempts to satisfy the following requirements:
 
-* Universal - server and client side
```

Comment:
This line shouldn't be removed. Just curious as to why you thought it should be removed?

Reply:
I didn't really think enough about React being fine to render on the server side. I did it because I thought that was one of the issues we were having with the combined library, but I didn't really think enough about it. My bad.

Reply:
Cool, no worries :) Just wanted to make sure I didn't give you the impression it was no longer universal